app/components/public/ticket-list.js

```diff
@@ -25,6 +25,36 @@ export default Component.extend(FormMixin, {
     applyPromotionalCode() {
       this.onValid(() => {
       });
+    },
+    updateOrder(ticket, count) {
+      let order = this.get('order');
+      ticket.set('orderQuantity', count);
+      order.set('amount', this.get('total'));
+      if (count > 0) {
+        order.tickets.addObject(ticket);
+      } else {
+        if (order.tickets.includes(ticket)) {
+          order.tickets.removeObject(ticket);
+        }
+      }
+    },
+    placeOrder() {
+      let order = this.get('order');
+      let event = order.get('event');
+      order.tickets.forEach(ticket => {
+        let attendee = ticket.orderQuantity;
+        let i;
+        for (i = 0; i < attendee; i++) {
```

Comment:
Do we have some sort of `forEach` loop in JavaScript like we have in Java ?  This `let i` doesn't look good.

Reply:
Sorry, this is the quantity. Please rename attendee to numberOfAttendees or equivalent.

Reply:
@srv-twry `forEach` is used to iterate over array. We don't have array here.